doc/source/whatsnew/v1.2.0.rst

```diff
@@ -331,6 +331,8 @@ Numeric
 - Bug in :class:`Series` where two :class:`Series` each have a :class:`DatetimeIndex` with different timezones having those indexes incorrectly changed when performing arithmetic operations (:issue:`33671`)
 - Bug in :meth:`pd._testing.assert_almost_equal` was incorrect for complex numeric types (:issue:`28235`)
 - Bug in :meth:`DataFrame.__rmatmul__` error handling reporting transposed shapes (:issue:`21581`)
+- Bug in :func:`select_dtypes` different behaviour on windows and linux for ``select_dtypes(include="int")`` (:issue:`36569`)
```

Comment:
```
- Bug in :func:`select_dtypes` different behavior between Windows and Linux with ``include="int"`` (:issue:`36569`)
```

Reply:
Ok

Reply:
The changes in this pull request will create problems with the existing pandas behavior, as you can see in the [comment above](https://github.com/pandas-dev/pandas/pull/36808#discussion_r498951543). Do you have any suggestions on how should proceed with this pull request?